### ISSUE TITLE
Nothing happens when clicking the "Firefox Monitor" and "Privacy Policy" hyperlinks from doorhangers

### ISSUE THREAD

**Issue by Softvision-CristinaBadescu** [mozilla/blurts-addon#51]
[Affected versions]:
- Firefox Beta 61.0b14
- Firefox Monitor 0.01, Branch 2, Branch 3, Branch 5

[Affected Platforms]:
- All Windows

[Prerequisites]:
- Have a clean Firefox profile with the latest Firefox Monitor v0.01 (custom built on 2018-06-20) installed.

[Steps to reproduce]:
1. Open the Firefox browser with the profile from prerequisites.
2. Navigate to a breached website ( eg. https://www.adobe.com).
3. Click the "Firefox Monitor" hyperlinked string and observe the behavior.

[Expected result]:
- You are redirected to the Firefox Monitor Landing Page with the breached website.

[Actual result]:
- Nothing happens.

[Notes]:
- This issue is also reproducible with the "Privacy Policy" hyperlinked string.
- This issue is not reproducible on Firefox Release 60.0.2.
- Here is a screen recording with this issue:
![firefox monitor - hyperlinked elements don t work](https://user-images.githubusercontent.com/31244235/41648767-85aa6b56-7482-11e8-9e6e-de4e66d01c41.gif)

**Comment by pdehaan** [comment]
Possibly similar/related to #47. Did you see any errors in your Browser Console?

**Comment by Softvision-CristinaBadescu** [comment]
Indeed, I get the following error in Browser Console "Error: Required argument triggeringPrincipal missing within openUILinkIn", therefore, this issue could have the same root cause as #47. I've closed this by mistake, but I'll reopen it and verify if the fix for #47 also fixes this issue.
![error browser console](https://user-images.githubusercontent.com/31244235/41717756-714a3632-7563-11e8-9338-f1fa79043e3b.png)

**Comment by Softvision-CristinaBadescu** [comment]
This issue is reproducible using the latest version of the "Firefox Monitor" add-on (0.01 custom built on 27.06.2018), both on Firefox Release 61.0 and Firefox Beta 62.0b3 unbranded builds.
Considering this, I'm going to reopen the issue and update the title.
![breach alerts - issue 51 still reproducible](https://user-images.githubusercontent.com/31244235/41981955-d99d8ada-7a32-11e8-904b-848254b08aef.gif)

**Comment by Softvision-CristinaBadescu** [comment]
Tested this with the latest Breach Alerts add-on (custom build on 06/27/2018) and the issue is still partially reproducible. After clicking the "Firefox Monitor" hyperlinked string, a new tab is opened but it's blank.
![breach alerts - 51](https://user-images.githubusercontent.com/31244235/41985992-849f8974-7a3d-11e8-816c-1035ee1bb660.gif)

**Comment by pdehaan** [comment]
That "/?breach=undefined" in the URL is highly suspicious.

**Comment by Softvision-CristinaBadescu** [comment]
This issue is no longer reproducible using the latest "Firefox Monitor" add-on (0.01 custom built on 28.06.2018), on Firefox Release 61.0 unbranded build. Verified on Windows 7 x64.
However, for certain websites, I'm also encountering the behavior described in #44.
